src/components/SourceTabs.js

```diff
@@ -249,7 +250,10 @@ const SourceTabs = React.createClass({
           pretty: isPretty({ url: source.get("url") })
         }),
         key: source.get("id"),
-        onClick: () => selectSource(source.get("id")),
+        onClick: () => {
+          selectSource(source.get("id"));
+          this.props.highlightSource(source.get("url"));
```

Comment:
do we need to call `highlightSource`? perhaps when we select the source, we can check to see if the source is expanded and if so highlight it.

One thing to note is that we don't need to expand directories to highlght the source. This is kinda an optional thing, if the source is shown, we'll highlight it.